src/GovUk.Education.ExploreEducationStatistics.Content.Api/Services/MethodologyService.cs

```diff
@@ -36,11 +37,12 @@ public class MethodologyService : IMethodologyService
 
         public async Task<Either<ActionResult, MethodologyViewModel>> GetLatestMethodologyBySlug(string slug)
         {
-            // TODO SOW4 EES-2375 lookup the MethodologyParent by slug when slug is moved to the parent
-            // For now, this does a lookup on the parent via any Methodology with the slug
             return await _persistenceHelper
-                .CheckEntityExists<Methodology>(query => 
-                    query.Where(mv => mv.Slug == slug))
+                .CheckEntityExists<Methodology>(
```

Comment:
Shouldn't this just be `.CheckEntityExists<MethodologyParent>(query => query.Where(mp => mp.Slug == slug))`?

We only need the MethodologyParent id to then call the repo below to get the latest published version, so don't need a methodology here or any includes.

Reply:
This comment looks like it was missing so I've addressed it now and updated the unit tests.